packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -158,10 +164,69 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
         await deployContract(mediumSizeContract, accounts[0].wallet);
 
         const remainingHbarsAfter = Number(await metrics.get(testConstants.METRICS.REMAINING_HBAR_LIMIT));
-        const expectedCost = 354819247;
+        const expectedCost = 331926551;
         verifyRemainingLimit(expectedCost, remainingHbarsBefore, remainingHbarsAfter);
       });
 
+      it('HBAR limiter is updated within acceptable tolerance range in relation to actual spent amount by the relay operator', async function () {
+        const TOLERANCE = 0.01;
```

Comment:
1% seems like a lot, can we try to reduce this to 0.1%?

```suggestion
        const TOLERANCE = 0.001;
```

Reply:
I've noticed that the actual variance is currently around 0.01, but sometimes it is even greater. I set it to 0.02 to reduce flakiness, but as we make further improvements we should be able to safely reduce it to 0.001 or even 0.